setup.cfg

```
@@ -33,7 +33,32 @@ install_requires =
 	torchvision>=0.9.0
 	tqdm>=4.59.0
 	uvicorn[standard]>=0.13.4
-	waitress>=2.0.0
+
+[options.extras_require]
+testing =
+	coverage
+	pytest>=4.0.0
+	pytest-cov>=2.5.1
+	pytest-xdist>=1.22.2
+	requests-mock
+lint =
```

Is this more descriptive?
```suggestion
dev =
```